users/admin.py

```
@@ -112,36 +116,75 @@ def make_unsubscribed(
+            envelope = build_envelope(
+                user_id=user_id, requested_by=requester_id
+            )
+            # StatsRefreshRequest 먼저 생성 (consumer 가 mark_processing 할 수 있도록)
+            lifecycle.mark_queued(
+                request_id=envelope["requestId"],
+                user_id=user_id,
+                requested_by=requester_id,
+            )
+            try:
+                service.redis_client.enqueue_message(envelope)
+            except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)